### DISCUSSION TITLE
NFC not producing vibration/beeps acknowledgment.

### DISCUSSION THREAD

**Discussion by vpv1952** [NightscoutFoundation/xDrip#3430] [Q&A]
I updated my POCO X5 PRO G5 from MIUI 14 to HyperOS 1.0.3.  

after that, xdrip seems to work fine, except one annoying detail:

-  when you NFC scan a libre 2 sensor, it doesn't produce a vibration/beep.  however the scan is successful.

Anyone has some hint for how to get those vibrations/beeps back?

thank you.
Virgílio

**Comment by PhilTU** [comment]
Do you get any vibration from any app ? I read that turning off haptic feedback also disables vibrate in hyperOS is it possible you did this ?

**Reply by vpv1952** [reply]
Yes, I had haptic feedback disabled.

, following your tip, i've enabled it and now it works:  NFC scanning gets the vibration!

by the way, xdrip NFC scanning still does not get any beep, but, as I didn't use this xdrip setting, I am not sure if it hyperOs fault, or if it was the MIUI that this phone had previously.

anyway, thank you for your help
Virgílio

⁣Get TypeApp for Android ​
